### ISSUE TITLE
Allow name edits

### ISSUE THREAD

**Issue by ehsanmasdar** [HackGT/ground-truth#20]
We've received several requests for name updates. It would be nice if this could be self-serve from the login portal.

**Comment by evan10s** [comment]
Need to be careful with this; allowing arbitrary name changes (especially last name) could let someone effectively transfer their acceptance to another person without us knowing.

**Comment by evan10s** [comment]
I agree that having a way to do this outside the DB would be nice, but the functionality for non-admins should be limited
